default/stringtables/en.txt

```diff
@@ -7768,6 +7768,12 @@ Derelict Fuel
 EFFECT_BIOWEAPON
 %planet% has lost %rawtext% population due to a %shippart% attack.
 
+EFFECT_BIOWEAPON_GAIA
+%planet% has lost %rawtext% population and [[special GAIA_SPECIAL]] due to a %shippart% attack.
```

Comment:
shouldn't say it lost Gaia unless it had it

Reply:
@geoffthemedio  It looks like I should use separate effect group or `If` with condition on Gaia special. Before it wasn't mentioned.

Reply:
> separate effect group

sounds reasonable...

Reply:
@geoffthemedio Splitted removing Gaia special.